github-status/handler.go

```diff
@@ -328,3 +328,12 @@ func hmacEnabled() bool {
 func buildStatus(status string, desc string, context string, url string) *github.RepoStatus {
 	return &github.RepoStatus{State: &status, TargetURL: &url, Description: &desc, Context: &context}
 }
+
+func truncate(maxLength int, message string) (string, bool) {
```

Comment:
Can you connect this to the output of the get logs function so that we don't post too much data to github?

Reply:
What is the upper limit?

Reply:
The maximum size is 64 KB.

https://developer.github.com/v3/checks/runs/#create-a-check-run